info/v1/container.go

```diff
@@ -295,10 +295,24 @@ type CpuCFS struct {
 	ThrottledTime uint64 `json:"throttled_time"`
 }
 
+// Cpu Aggregated scheduler statistics
+type CpuSchedstat struct {
+	// https://www.kernel.org/doc/Documentation/scheduler/sched-stats.txt
+
+	// These numbers are summed up over all processes in a cgroup and can therefore not capture processes after they are gone.
+	// time spent on the cpu
+	RunTime uint64 `json:"run_time"`
```

Comment:
Is this different from CPU usage?

Reply:
I am not so sure what the difference is, but they look quite distinct:
![screenshot from 2018-01-31 15-25-40](https://user-images.githubusercontent.com/8849712/35628224-811c8628-069b-11e8-8db0-7df78b62f7eb.png)

Reply:
Since CPU and RunTime are the same, we should omit it from the API.

Reply:
@nielsole I don't think those graphs are comparable, when summing up those container stats there's some double accounting going on. I'd recommend doing it for a single container.

Reply:
The two graphs here are not comparable, but he says here https://github.com/google/cadvisor/pull/1872#issuecomment-361949155 that he figured it out and they are the same.

Reply:
Actually I did not figure it out. #1872 was referring to a problem I had with running cAdvisor.

Reply:
What is the resolution here @nielsole?  Is this the same as cpu usage, or are they different metrics?